recipes/cuda-python/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "cuda-python" %}
+{% set major_version = "11" %}
+{% set minor_version = "4" %}
+{% set patch_version = "0" %}
+{% set version = major_version ~ "." ~ minor_version ~ "." ~ patch_version %}
```

Comment:
For the bot to work, you need to do this the other way. `set version = "11.4.0"` and then `major_version = version.split(".")[0]`

Reply:
Updated.